### PR TITLE
tts: support more languages in playht and elevenlabs

### PR DESCRIPTION
**Pull request by aconchillo**
#### Please describe the changes in your PR. If it is addressing an issue, please reference that as well.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to **Pipecat** will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Add support for more languages to ElevenLabs (arabic, croatian, filipino,
```
Comment (markbackman) [inline CHANGELOG.md:12]:
```suggestion
- Add support for more languages to ElevenLabs (Arabic, Croatian, Filipino,
```

Comment (markbackman) [reply CHANGELOG.md:12]: Let's capitalize all of these languages.

**Review by markbackman — APPROVED** [review]
LGTM! Arabic support in 11Labs will be nice. Lots of demand there.

File: CHANGELOG.md (hunk 1 above)
Comment (aconchillo) [reply CHANGELOG.md]: Done!

**Pull request merged**